ast-visual-studio-extension/CxWrapper/CxWrapper.cs

```diff
@@ -384,6 +384,30 @@ public List<CodeBashing> CodeBashingList(string cweId, string language, string q
             return JsonConvert.DeserializeObject<List<CodeBashing>>(codebashingLink);
         }
 
+        /// <summary>
+        /// Learn More and Code Samples
+        /// </summary>
+        /// <param name="queryId"></param>
+        /// <returns></returns>
+        public List<LearnMore> LearnMoreAndRemediation(string queryId)
+        {
+            logger.Info("getting learn more and remediation examples");
```

Comment:
Move this message to the CxConstants like we are doing in other calls. Upper case the first letter please